networking_cisco/plugins/cisco/device_manager/plugging_drivers/aci_vlan_trunking_driver.py

```diff
@@ -233,12 +236,22 @@ def extend_hosting_port_info(self, context, port_db, hosting_device,
             hosting_info['cidr_exposed'] = ext_dict['cidr_exposed']
             hosting_info['gateway_ip'] = ext_dict['gateway_ip']
         else:
+            router_id = port_db.device_id
+            router = self.l3_plugin.get_router(context, router_id)
+            # skip routers not created by the user -- they will have
+            # empty-string tenant IDs
+            if router.get(ROUTER_ROLE_ATTR):
+                return
+            snat_ips = self.apic_driver.get_snat_ip_for_vrf(context,
+                router['tenant_id'], net)
```

Comment:
I think router['tenant_id'] represents vrf_id here. But do we have to consider all those different modes (single_tenant_mode, per_tenant_context, ...etc)? For example what happens if totally there is only 1 global VRF but looks like the code here will still use different tenant_id....